openapi.json

```diff
@@ -266,6 +266,131 @@
         ]
       }
     },
+    "/clusters/reports/": {
```

Comment:
Should it be here `orgId` as well?

Reply:
unfortunatelly ACM has no way (right now) how to provide orgId :/

Reply:
This looks like a big flaw in the authorization. This endpoint can be used to access data that are not authorized for the requester.